### ISSUE TITLE
I sincerely hope that visual prompts will be added to the Textstudio editor.

### ISSUE THREAD

**Issue by Wangzai-hub** [texstudio-org/texstudio#3692]
### Describe the feature and the current behavior/state
It is well known that proper indentation can make documents look clearer, especially when editing formulas. Currently there is no indentation prompt in the textstudio editor, and when I press the delete key, it only removes one space instead of the entire indentation. I sincerely hope that the Textstudio editor will add visual prompts, and that each delete will remove four spaces, which is an indentation, instead of just one space.
![2](https://github.com/texstudio-org/texstudio/assets/114059567/f3216c23-07eb-4f49-9baf-1d4de5f557f6)
### Who will benefit with this feature?
everyone

**Comment by dbitouze** [comment]
It would also be easier to find your way around nested lists.

**Comment by sunderme** [comment]
by default txs places "tabs" as such.
You can set txs to use spaces instead but that is not what you want: options/editor/replace indenation tab by spaces.
If a tab is used, that is removed as a single char.
The tab width can be set in option/adv. editor/tab width.
There you can also set "show spaces" which shows tabs as an grey arrow.

**Comment by Wangzai-hub** [comment]
> by default txs places "tabs" as such. You can set txs to use spaces instead but that is not what you want: options/editor/replace indentation tab by spaces. If a tab is used, that is removed as a single char. The tab width can be set in option/adv. editor/tab width. There you can also set "show spaces" which shows tabs as an grey arrow.

Thank you for your attention to my request. As you describe, I have changed the settings in my Textstudio and now I use tab as an indentation. This can indeed be used as a temporary method. 
![Uploading 1.png…]()
But I prefer four spaces with visual prompts as a better way to indent. Also, when I press Delete, the four spaces are removed, just like in most other code editors.
I wonder if this will happen in future versions.

**Comment by sunderme** [comment]
I don't think that it is really a goal for texstudio to emulate programming editors on this.
For changing indentation, tab/shift+tab works well. Tabs as characters are totally fine.
And finally, on the left border, txs shows on hover foldable/connected regions.

That needs to suffice.

**Comment by Wangzai-hub** [comment]
> I don't think that it is really a goal for texstudio to emulate programming editors on this. For changing indentation, tab/shift+tab works well. Tabs as characters are totally fine. And finally, on the left border, txs shows on hover foldable/connected regions.
> 
> That needs to suffice.

Okay, thank you for your answer.

**Comment by samcarter** [comment]
Possibly a duplicate of https://github.com/texstudio-org/texstudio/issues/1309 ?
